plugins/action_generate_review_assignment_records/plugin.json

```diff
@@ -3,15 +3,9 @@
   "code": "generateReviewAssignments",
   "name": "Generate Review Assignment Records",
   "description": "Generates review_assignment records for current application",
-  "required_parameters": [],
-  "optional_parameters": ["applicationId", "reviewId", "isReview"],
-  "output_properties": [
-    "reviewAssignmentIds",
-    "reviewAssignmentAssignerJoins",
-    "reviewAssignmentAssignerJoinIds",
-    "currentReviewLevel",
-    "nextReviewLevel"
-  ],
+  "required_parameters": ["userId"],
+  "optional_parameters": [],
+  "output_properties": ["updatedApplications", "reviewAssignments"],
```

Comment:
Oh, I think that was the wrong action updated. No?

Reply:
Yes, sorry. When I merged this into your branch I used your one instead of this to resolve the conflict, so should be all good over there.